### PR TITLE
[Community Pipeline]  Rerender-A-Video: Zero-Shot Video-to-Video Translation

### PR DESCRIPTION
**Pull request by SingleZombie**
# What does this PR do?

Related issue: #5467  

The PR is the Diffusers implementation of zero-shot video-to-video translation pipeline [Rerender_A_Video](https://github.com/williamyang1991/Rerender_A_Video).

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [x] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [x] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/c/discussion-related-to-httpsgithubcomhuggingfacediffusers/63)? Please add a link to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/diffusers/tree/main/docs#writing-source-documentation).
- [x] Did you write any new necessary tests?

## Who can review?

 @patrickvonplaten @sayakpaul

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
Hey @SingleZombie, 

Can you run `make style` to make sure the quality checks pass?

**Comment by SingleZombie** [comment]
Hi @patrickvonplaten, I have formatted the codes.

File: examples/community/rerender_a_video.py [hunk 1]
```diff
@@ -0,0 +1,1175 @@
...
+        Returns:
+            [`~pipelines.stable_diffusion.StableDiffusionPipelineOutput`] or `tuple`:
+            [`~pipelines.stable_diffusion.StableDiffusionPipelineOutput`] if `return_dict` is True, otherwise a `tuple.
+            When returning a tuple, the first element is a list with the generated images, and the second element is a
+            list of `bool`s denoting whether the corresponding generated image likely represents "not-safe-for-work"
+            (nsfw) content, according to the `safety_checker`.
+        """
+        controlnet = self.controlnet._orig_mod if is_compiled_module(self.controlnet) else self.controlnet
+
+        flow_model = GMFlow(
```
Comment (a-r-r-o-w) [inline examples/community/rerender_a_video.py:694]: Can we move this out of the `__call__` function? I'm not sure why we would reinitialize model on every inference call...

**Review by a-r-r-o-w — COMMENTED** [review]
Hi @SingleZombie. Thank you so much for adding this! Just wondering if it's ready to use or not yet...

**Comment by SingleZombie** [comment]
Hi @a-r-r-o-w. I have edited the codes.  The pipeline is ready to use. I have tested it on my machine.

**Comment by a-r-r-o-w** [comment]
Thank you so much for your amazing work @SingleZombie! I'm still trying to fully understand your paper and it is very informative and intuitive. Your code here helps a lot for the same :heart:

**Comment by patrickvonplaten** [comment]
Great job @SingleZombie

**Pull request merged**